packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -23,6 +27,7 @@ export interface GridFilterFormProps {
   showMultiFilterOperators?: boolean;
   multiFilterOperator?: GridLinkOperator;
   disableMultiFilterOperator?: boolean;
+  toFocus?: { value: boolean };
```

Comment:
To keep with the convention: https://github.com/mui-org/material-ui-x/blob/0fd81307c6ffdc1031726680a5f302bb280058c4/packages/grid/_modules_/grid/components/cell/GridCell.tsx#L27

```suggestion
  hasFocus?: { value: boolean };
```